cocos/renderer/gfx-metal/MTLUtils.mm

```diff
@@ -1626,5 +1630,16 @@ void main() {
     renderPassDescriptor.colorAttachments[slot].loadAction = MTLLoadActionLoad;
 }
 
+void mu::clearUtilResource() {
+    if(!renderPassMap.empty()) {
+        for (auto pass : renderPassMap) {
+            //TODO: create and destroy not in the same level
+            pass.second->destroy();
+            delete pass.second;
```

Comment:
Please use CC_DELETE as we may override delete and use our own memory management mechanism.